modules/local/gatk4/printreads/main.nf

```diff
@@ -0,0 +1,56 @@
+process GATK4_PRINTREADS {
```

Comment:
Is it within local modules to avoid the wait of making it into a nf-core module or is there any other reason ?

Reply:
Yupe, I have a PR open to add it to nf-core modules.